pkg/emulator/libretro/repo/github/repository.go

```diff
@@ -0,0 +1,20 @@
+package github
+
+import (
+	"github.com/giongto35/cloud-game/v2/pkg/emulator/libretro/core"
+	"github.com/giongto35/cloud-game/v2/pkg/emulator/libretro/repo"
+	"github.com/giongto35/cloud-game/v2/pkg/emulator/libretro/repo/buildbot"
+)
+
+type Repo struct {
+	buildbot.Repo
+}
+
+func NewGithubRepo(address string, compression string) Repo {
+	return Repo{Repo: buildbot.NewBuildbotRepo(address, compression)}
```

Comment:
the github repo is now mocked by buildbot ?

Reply:
Yes. By that, I meant exact same file structure as in buildbot. It's a mirror with free GitHub hosting (:

Reply:
I'll create one later.

Reply:
If you think about that, it's free, versioned, tagged (with branches), replicated (with forks) (:

Reply:
it's ok for now, I think let's keep buildbot. I was wondering if repo can be some storage like s3, or the google cloud storage like currently.

Reply:
You mean with a client (login:pass). Haven't worked yet with cloud storage providers but it's easy to add now. In any case, it needs some background management (copy buildbot).
I assume it's related more to games repo which is yet to be implemented. For that, yes, we will need private storage.